### PR TITLE
(feat) : KHP3-6501 centralize commodity and billable services under one model

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
This PR consolidate how we manage billable services and commodity items i.e Pharms & Non Pharms stock items to one centralized dashboard. In this PR the following adjustment have been made.

1. Depreciated the old method of managing billable items using purchase price from stock model.
2. Add new table that display add billable items i.e stock items and service in one place
3. Add forms for adding and editing services and commodity prices

Pending item
1. Deleting billable items is still an issue, the backend fails with error `404` cc @ojwanganto @makombe @patryllus to assist address the issue.
2. Add unit test to prevent regression of this feature

## Screenshots
Kapture 2024-09-04 at 00 06 06

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/billable-services/billables/billable-summary-table.component.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+                ))}
+              </TableBody>
+            </Table>
+          </TableContainer>
+        )}
+      </DataTable>
+      <Pagination
+        backwardText="Previous page"
+        forwardText="Next page"
+        itemsPerPageText="Items per page:"
```
Comment (makombe) [inline packages/esm-billing-app/src/billable-services/billables/billable-summary-table.component.tsx:193]: We can also make this translatable.

**Review by makombe — DISMISSED** [review]
Thanks @donaldkibet  LGTM

File: packages/esm-billing-app/src/billable-services/billable-services.component.tsx [hunk 2]
```diff
@@ -1,323 +1,28 @@
...
-        />
-      )}
-    </>
+    <Tabs>
+      <TabList aria-label={t('priceManagement', 'Price management')} contained>
+        <Tab
+          className={styles.tabHeader}
+          renderIcon={Task}
+          secondaryLabel={t('billableDescription', 'Billable service and commodity prices')}>
+          {t('Services', 'Services')}
```
[the remark this answers is not in the document]
Comment (ojwanganto) [reply packages/esm-billing-app/src/billable-services/billable-services.component.tsx:17]: I think we need to change this to something like 'Billables'. I am not sure if there is a better name we can use

[the remark this answers is not in the document]
Comment (donaldkibet) [reply packages/esm-billing-app/src/billable-services/billable-services.component.tsx]: Gone with 
`Medical Invoice Items` for the title and for description `Billable services and consumable prices`

**Review by ojwanganto — APPROVED** [review]

**Pull request merged**